meson.build

```diff
@@ -113,6 +112,14 @@ warning_flags = [
 cc = meson.get_compiler('c')
 add_project_arguments(cc.get_supported_arguments(warning_flags), language : 'c')
 
+# use honggfuzz for parallel persistent fuzzing
+if cc.has_function('HF_ITER')
+  honggfuzz = find_program('honggfuzz')
+  conf.set('HAVE_HF_ITER', '1')
+else
+  honggfuzz = find_program('honggfuzz', required : false)
```

Comment:
I don't know if this works, but can you have:
`honggfuzz = find_program('honggfuzz', required : cc.has_function('HF_ITER')`

Reply:
I still need the HAVE_HF_ITER define tho

Reply:
Like this maybe: 
```
honggfuzz = find_program('honggfuzz', required : cc.has_function('HF_ITER')
if honggfuzz.found()
   conf.set('HAVE_HF_ITER', '1')
endif
```

Reply:
Works for me! Fixed.